### ISSUE TITLE
ErrorType.create unnecessarily throws on certain codes

### ISSUE THREAD

**Issue by meditativeape** [palantir/http-remoting-api#47]
Why does `ErrorType.create` throws on `CUSTOM_CLIENT` and `CUSTOM_SERVER`?
https://github.com/palantir/http-remoting-api/blob/develop/errors/src/main/java/com/palantir/remoting/api/errors/ErrorType.java#L103

**Comment by uschi2000** [comment]
looks like this behavior is obsolete now, it's functionally equivalent to the server/client methods, right?

**Comment by meditativeape** [comment]
Yes, so I think it doesn't need to throw any more? Reason I ask is Conjure generated code uses `ErrorType.create` for all error codes, and fails at runtime if any is CUSTOM_CLIENT or CUSTOM_SERVER. Seems easier to change in remoting-api than special case them in Conjure.

**Comment by uschi2000** [comment]
Agreed.
